### PR TITLE
Typo: ocr-capabilites -> ocr-capabilities

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: hocr-check [hunk 1]
```diff
@@ -79,7 +79,7 @@ doc = html.parse(args.file)
 
 # check for presence of meta information
 test_ok(doc.xpath("//meta[@name='ocr-system']")!=[], "//meta[@name='ocr-system']")
-test_ok(doc.xpath("//meta[@name='ocr-capabilites']")!=[], "//meta[@name='ocr-capabilites']")
+test_ok(doc.xpath("//meta[@name='ocr-capabilites']")!=[], "//meta[@name='ocr-capabilities']")
```
Comment (kba) [inline hocr-check]: I think you're only correcting the typo in the comment, not the xpath itself.

Comment (zuphilip) [inline hocr-check]: Uh, yes, I will add the other one as well...

**Comment by kba** [comment]
Looks good to me.

**Comment by zuphilip** [comment]
Travis is not happy: https://travis-ci.org/UB-Mannheim/hocr-tools/builds/181636208

**Comment by zuphilip** [comment]
Okay, the test cases also have the typo. I will fix them as well...

**Comment by zuphilip** [comment]
Okay, I fixed the error on some more test cases. Moreover, I added a new test "notok-typo" to test exactly this case.

**Pull request merged**